### PR TITLE
ENH extra symlink

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,6 +27,8 @@ outputs:
       - llvm-tools {{ version }}
       - cctools_{{ cross_target_platform }}
       - ld64_{{ cross_target_platform }}
+      # this brings in an extra symlink of ld -> x86_64-apple-darwin13.4.0-ld
+      - ld64  # [cross_target_platform == "osx-64"]
```
Comment (isuruf) [inline recipe/meta.yaml:31]:
```suggestion
      - ld64  # [cross_target_platform == target_platform]
```

Comment (isuruf) [reply recipe/meta.yaml:31]: Can you also add `cctools` ?

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by isuruf** [comment]
Can you also add cctools ?

**Comment by beckermr** [comment]
ready for another look @isuruf !

**Comment by beckermr** [comment]
Thanks @isuruf !

**Pull request merged**